README.md

```diff
@@ -1,28 +1,52 @@
 # vosk-cli
 
-This python package serves as an Vosk interface for Opencast. It allows to generate subtitles (WebVVT files) from Video and Audio sources via Vosk.
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/elan-ev/vosk-cli/Build%20and%20publish)
+![GitHub](https://img.shields.io/github/license/elan-ev/vosk-cli)
+![PyPI](https://img.shields.io/pypi/v/vosk-cli)
+
+This python package serves as an Vosk interface for Opencast. It allows to generate subtitles (WebVTT files) from Video and Audio sources via Vosk.
 
 ## Installation
 
 ### 1. Install vosk-cli
+To install the [latest stable version of vosk-cli](https://pypi.org/project/vosk-cli/), run
+
+```
+pip install vosk-cli
+```
 
-Clone this project and run
+Alternatively, to install the latest development version, clone this project and inside the project directory run
 
 ```
-python setup.py install
+pip install .
 ```
 
 ### 2. Install dependencies
 
-- FFmpeg
-- `pip install -r requirements.txt`
+vosk-cli uses ffmpeg to preprocess input files. The easiest way to install ffmpeg is by using a package manager.
+To install ffmpeg on **Linux**, run
+
+```
+sudo apt install ffmpeg
```

Comment:
This wouldn't work for me. I'm on fedora and my default package manager is `dnf`. On Gentoo, it would be something else, on Arch, it would be something else again. It's basically impossible to list them all :D

That's why I would probably not try to list all options, but just tell users to “install FFmpeg” and potentially link http://ffmpeg.org/download.html